src/pages/Auth/FindPasswordPage/Complete/index.tsx

```diff
@@ -1,23 +1,49 @@
+import LogoIcon from 'assets/svg/Login/logo.svg';
+import CheckIcon from 'assets/svg/orenge-check.svg';
 import { useNavigate } from 'react-router-dom';
 import ROUTES from 'static/routes';
+import useMediaQuery from 'utils/hooks/layout/useMediaQuery';
+import styles from './Complete.module.scss';
 
 function CompletePage() {
   const navigate = useNavigate();
+  const isMobile = useMediaQuery();
 
-  const goToLogin = () => {
-    navigate(ROUTES.Auth());
-  };
+  const goToLogin = () => navigate(ROUTES.Auth());
+
+  const title = '비밀번호 변경 완료';
```

Comment:
해당 컴포넌트에서만 사용하는 상수라면 컴포넌트 바깥에서 전역적으로 선언해서 사용해도 좋을 거 같아요.
그리고 대문자로 쓰는 게 컨벤션에 맞을 거 같습니다!
```suggestion
  const TITLE = '비밀번호 변경 완료';
```